cmd/gatekeeper/main.go

```diff
@@ -13,6 +13,7 @@ import (
 const (
 	skipLabel      = "skip/ci"
 	doNotMergeHold = "do-not-merge/hold"
+	e2eTestConfigFile    = "./tests/e2e/config.yaml"
```

Comment:
Not sure why this didn't auto-format 🤔 
```suggestion
	skipLabel                  = "skip/ci"
	doNotMergeHold     = "do-not-merge/hold"
	e2eTestConfigFile    = "./tests/e2e/config.yaml"
```